contributing/lts-release.md

```diff
@@ -1,6 +1,8 @@
 # Long-Term Stable (lts) Release Channel
 
-The **lts** release channel provides a previous **stable** build for one-month intervals. Approximately monthly, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
+The **lts** release channel provides a previous **stable** build for one-month intervals. On the second Monday of each month, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel.
+
+In the event that the second Monday of the month falls on a holiday, the promotion will be performed the following day (ie. Tuesday) instead.
```

Comment:
This copy is replicated in `release-schedule.md`. @CrystalOnScript would love your opinion on this.

Reply:
"or is during a code freeze" somewhere in here

Reply:
Also, which holidays? AMP is international :)

Reply:
Updated; PTAL